src/engine/renderer/tr_shader.cpp

```diff
@@ -5572,6 +5572,88 @@ bool CheckShaderNameLength( const char* func_err, const char* name, const char*
 	return true;
 }
 
+static bool CheckStage( shaderStage_t *pStage )
+{
+	struct stageCheck_t {
+		bool expected;
+		bool check;
+		bool fallback;
+		std::string name;
```

Comment:
Could be a `const char *` (to avoid allocations)

Reply:
Done.